component-samples/components-oauth/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>OAuth with ArcGIS Map Components</title>
+    <link rel="icon" href="data:;base64,=" />
+  </head>
+
+  <body>
+    <arcgis-map item-id="d5dda743788a4b0688fe48f43ae7beb9">
```

Comment:
Should there be a note here or in the README, to say this should be replaced with a secure web map item id (or a web map with secure layers) in the portal set in main.js?

Reply:
Updated README with an "Important" message to add the note.